challenges/objects-arrays.js

```diff
@@ -100,22 +136,31 @@ The zoos need a list of all their animal's names (animal_name only) converted to
 */
 
 const lowCaseAnimalNames = [];
+zooAnimals.map((aniLow) => {
```

Comment:
Remember that the .map() method creates a new array for you by default. No pushing!